JASP-Engine/JASP/R/LScommon.R

```diff
@@ -15,6 +15,21 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
+# developmental stuff
+RDEBUG <- function(message){
```

Comment:
Running this while in JASP messes up the logging

Reply:
yeah, sorry, forgot to delete this